README.md

```diff
@@ -16,6 +16,26 @@ This is a dictionary which can hold key/value pairs of all supported Postfix
 configuration parameters. Keys not supported by the installed Postfix are
 ignored.  The default is empty `{}`.
 
+You can specify `previous: "replaced"` within the `postfix_conf` variable to
```

Comment:
Don't use quotes if you don't need them - this should just be `previous: replaced` - here and below

Reply:
I did it for consistency because this role uses quotes everywhere. I will remove quotes from the code added in this PR, then will remove quotes everywhere else in a separate PR for consistency.